LICENSE

```diff
@@ -0,0 +1,99 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             BabylonLabs, Ltd.
+
+Licensed Work:        finality-provider
+                      The Licensed Work is (c) 2023 BabylonLabs, Ltd.
+
+Additional Use Grant: None.
```

Comment:
We typically add an additional use grant, please make sure that the license is compatible with the one here: https://github.com/babylonlabs-io/babylon/blob/main/LICENSE#L16

Reply:
Added